build_tools/tests/fileset_tool_test.py

```diff
@@ -8,6 +8,9 @@
 import tempfile
 import unittest
 
+sys.path.insert(0, os.fspath(Path(__file__).parent.parent))
```

Comment:
I usually do Path(__file__).resolve()

Reply:
That is what I also tried but without _any_ luck. When only using `Path`, without `os` in the one or other way, it always failed with
> ModuleNotFoundError: No module named '_therock_utils'
for me.